client/src/assets/icons/bluesky-logo-icon.tsx

```diff
@@ -0,0 +1,25 @@
+export function BlueskyLogoIcon({ className }: { className?: string }) {
```

Comment:
We can improve the support of additional properties if needed with `props: SVGProps<SVGSVGElement>`